iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/BranchData.java

```diff
@@ -222,7 +222,7 @@ private double getB1(TwoWindingsTransformer twt, boolean specificCompatibility)
     }
 
     private double getRho1(TwoWindingsTransformer twt) {
-        double rho = twt.getRatedU2() / twt.getRatedU1();
+        double rho = twt.getTerminal2().getVoltageLevel().getNominalV() / twt.getRatedU1();
```

Comment:
Rho1 is the transformer ideal ratio on side 1.
See http://www.powsybl.org/docs/loadflow/validation.html.
I think that rho1 = twt.getRatedU2() / twt.getRatedU1();

Reply:
Corrected.